iOS/MSQASignInSwift/MSQASignInCustomization.swift

```diff
@@ -0,0 +1,288 @@
+let MSQASignInButtonNormalBackgroundColorDark: Int = 0x2f2f_2fff
+let MSQASignInButtonNormalBorderColorLight: Int = 0xd6d6_d6ff
+let MSQASignInButtonNormalBorderColorDark: Int = 0x2f2f_2fff
+
+let MSQASignInButtonPressedBackgroundColorLight: Int = 0xefef_efff
+let MSQASignInButtonPressedBackgroundColorDark: Int = 0x2727_27ff
+let MSQASignInButtonPressedBorderColorLight: Int = 0xd6d6_d6ff
+let MSQASignInButtonPressedBorderColorDark: Int = 0x2727_27ff
+
+let MSQASignInButtonTextColorLight: Int = 0x3231_30ff
+let MSQASignInButtonTextColorDark: Int = 0xffff_ffff
+
+/// Types supported by Microsoft Quick Auth sign in button.
+@available(iOS 13.0, *)
+public enum MSQASignInButtonType: String, Identifiable, CaseIterable {
+  case standard
+  case icon
+  public var id: String { rawValue }
+}
+
+/// Types supported by Microsoft Quick Auth sign in button.
+@available(iOS 13.0, *)
+public enum MSQASignInButtonTheme: String, Identifiable, CaseIterable {
+  case light
+  case dark
+  public var id: String { rawValue }
+}
+
+/// Sizes supported by Microsoft Quick Auth sign in button.
+@available(iOS 13.0, *)
+public enum MSQASignInButtonSize: String, Identifiable, CaseIterable {
+  case large
+  case medium
+  case small
+  public var id: String { rawValue }
+}
+
+/// Texts supported by Microsoft Quick Auth sign in button.
+@available(iOS 13.0, *)
+public enum MSQASignInButtonText: String, Identifiable, CaseIterable {
+  case signInWith
+  case signUpWith
+  case continueWith
+  case signIn
+  public var id: String { rawValue }
+}
+
+/// Shapes supported by Microsoft Quick Auth sign in button.
+@available(iOS 13.0, *)
+public enum MSQASignInButtonShape: String, Identifiable, CaseIterable {
+  case rectangular
+  case rounded
+  case pill
+  public var id: String { rawValue }
+}
+
+/// Layouts supported by Microsoft Quick Auth sign in button.
+@available(iOS 13.0, *)
+public enum MSQASignInButtonLayout: String, Identifiable, CaseIterable {
+  case logoTextLeft
+  case logoLeftTextCenter
+  case logoTextCenter
+  public var id: String { rawValue }
+}
+
+/// States supported by Microsoft Quick Auth sign in button.
+@available(iOS 13.0, *)
+public enum MSQASignInButtonState: String, Identifiable, CaseIterable {
+  case normal
+  case pressed
+  public var id: String { rawValue }
+}
+
+@available(iOS 13.0, *)
+struct MSQASignInButtonStyle: ButtonStyle {
+  let type: MSQASignInButtonType
+  let theme: MSQASignInButtonTheme
+  let size: MSQASignInButtonSize
+  let text: MSQASignInButtonText
+  let shape: MSQASignInButtonShape
+  let layout: MSQASignInButtonLayout
+  let state: MSQASignInButtonState
+
+  public var font: Font {
+    return Font(uiFont)
+  }
+
+  private var uiFont: UIFont {
+    let calculatedFontSize = fontSize
+    if let registeredFont = UIFont(name: MSQASignInButtonFontName, size: calculatedFontSize) {
+      return registeredFont
+    }
+    guard
+      let fontURL = MSQASignInButtonHelper.url(
```

Comment:
Currently, for the implement of UIKit, we only adopt `SegoeUI-SemiBold` font for English, for other languages, we use the system default one, would you please align with that?
https://github.com/microsoft/quick-authentication-mobile/blob/c3e9dd3510b7d6f6f3b157bb21a773b54e7e512c/iOS/MSQASignIn/src/MSQASignInButton.m#L640-L644

Reply:
Resolved by using system default font for non-English languages.